frontend/src/bundles/home/pages/HomePage.vue

```diff
@@ -0,0 +1,264 @@
+            <v-col
+              cols="12"
+              md="6"
+            >
+              <v-row
+                align="center"
+              >
+                <v-col
+                  v-for="(item, index) in itemsAbout"
+                  :key="index"
+                  cols="12"
+                  md="6"
+                >
+                  <v-img
+                    max-width="37"
+                    max-height="37"
+                    height="100%"
+                    contain
+                    :src="item.icon"
+                  />
+                  <h4 class="h4 font-weight-bold pb-5">{{ item.title }}</h4>
+                  <p>{{ item.text }}</p>
+                </v-col>
+              </v-row>
+            </v-col>
+            <v-col
+              cols="12"
+              md="6"
+            >
+              <v-img
+                max-height="465"
+                max-width="855"
+                contain
+                src="@/assets/image/home_section_2.png"
+              />
+            </v-col>
+          </v-row>
+        </v-container>
+      </div>
+      <div
+        class="join-us pa-md-11 pa-6"
+        tag="section"
+      >
+        <v-container>
+          <v-row
+            align="center"
+          >
+            <v-col
+              cols="12"
+              md="7"
+            >
+              <v-img
+                max-height="500"
+                max-width="900"
+                contain
+                src="@/assets/image/home_section_3.png"
+              />
+            </v-col>
+            <v-col
+              cols="12"
+              md="5"
+            >
+              <h4 class="h4 font-weight-bold pb-5">
+                More than a just app
+              </h4>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum fermentum viverra amet id arcu. Id adipiscing risus pulvinar nisi, turpis.
+              </p>
+              <p>
+                Malesuada elit nullam sollicitudin ante enim non tempus.
+              </p>
+              <v-btn
+                class="text-capitalize font-weight-bold"
+                color="button-main"
+                large
+                raised
+                rounded
+                outlined
+                depressed
+                max-width="162"
+                width="100%"
+                to="/auth/register"
+              >
+                Join
+              </v-btn>
+            </v-col>
+          </v-row>
+        </v-container>
+      </div>
+    </v-main>
+    <FooterLandingNav />
+  </div>
+</template>
+
+<script>
+
+import HeaderLandingNav from '../components/HeaderLandingNav';
+import FooterLandingNav from '../components/FooterLandingNav';
+import namespace from '@/bundles/common/store/modules/user/namespace';
+import { mapState } from 'vuex';
+
+export default {
+  components: {
+    HeaderLandingNav,
+    FooterLandingNav,
+  },
+  data() {
+    return {
+      itemsAbout: [
```

Comment:
If data is static and will never change better to create a computed property for it

Reply:
fixed